pkg/trim-after-delete/main.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
```

Comment:
I would really be inclined to not vendor `docker/docker` at all, and just write a pure http version. The events protocol, especially given what you want from it, is pretty trivial I would think. Definitely less than 160,000 lines of code...